patches/4156084C - Transformers War for Cybertron.patch.toml

```diff
@@ -1,10 +1,10 @@
 title_name = "Transformers War for Cybertron"
 title_id = "4156084C" # AV-2124
 hash = "0D6CA1E3F19F0087" # default.xex
-media_id = [
-    "02619995", # Disc (USA): http://redump.org/disc/14254/
-    "3AFEC771" # Disc (Europe): http://redump.org/disc/67200/ 
-]
+#media_id = [
+#    "02619995", # Disc (USA): http://redump.org/disc/14254/
+#    "3AFEC771" # Disc (Europe): http://redump.org/disc/67200/ 
```

Comment:
It looks like line 6 ends with a trailing space, and the linter is complaining about it despite the line being commented out.

Reply:
That fixed it, thanks!